examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/formsportallink/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="AEM Forms Link Component for the Core Components Library"
+    jcr:primaryType="cq:Component"
+    jcr:title="AEM Forms Link"
```

Comment:
formsportal/link in the URL

Can you check with PM for the title and description of the proxy component ? I don't see forms portal in the title or the description

Reply:
Asked for inputs on JIRA. Will update PR whenever description is available